media-sound/sox/sox-14.4.2-r2.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Looks like 7 can be used. Please rebuild and test it with 7 as well, I see `ED` being used that can break with 6->7 bump.

Reply:
modified to use the portable version of ED var: https://dev.gentoo.org/~mgorny/articles/the-ultimate-guide-to-eapi-7.html#d-ed-root-eroot-no-longer-have-a-trailing-slash